coriolis/osmorphing/oracle.py

```diff
@@ -26,15 +26,15 @@ def _get_oracle_repos(self):
         repos = []
         major_version = int(self._version.split(".")[0])
         if major_version < 8:
-            self._yum_install(
-                ['yum-utils'],
-                self._find_yum_repos(['ol%s_latest' % major_version]))
-            # TODO(apilotti): for ULN users, use the corresponding repos
-            # e.g.: ol7_x86_64_addons
+            repo_file_path = (
+                '/etc/yum.repos.d/new-public-yum-ol%s.repo' % major_version)
+            self._exec_cmd_chroot(
+                "curl -L http://public-yum.oracle.com/public-yum-ol%s.repo "
```

Comment:
I'm assuming this would be equivalent to the `ol%s_latest` repo we `yum install`d previously, but `yum-utils` no longer gets installed.

Are we sure none of the providers expect something from `yum-utils` to be available?

Reply:
We don't modify yum in any way (as we should), so yum-utils is not needed for providers. Even if that was the case, this `yum-utils` installation only occured for OL guests. And yes, it is the same repo URL as the former.